v1alpha1apiserver/vendor/github.com/openshift/hive/pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -251,7 +251,7 @@ const (
 	InstallFailingCondition ClusterDeploymentConditionType = "InstallFailing"
 
 	// DNSNotReadyCondition indicates that the the DNSZone object created for the clusterDeployment
-	// (ie manageDNS==true) has not yet indicated that the DNS zone is successfully responding to queries.
+	// (ie managedDNS==true) has not yet indicated that the DNS zone is successfully responding to queries.
```

Comment:
I guess nobody else runs `make vendor` since this change has been here erroneously since https://github.com/openshift/hive/pull/980.